steps/delius/contact/create-contact.ts

```diff
@@ -0,0 +1,40 @@
+import { expect, Page } from '@playwright/test'
+import { findContactsByCRN } from './find-contacts'
+import { fillTime, selectOption } from '../utils/inputs'
+import { waitForAjax } from '../utils/refresh'
+
+export interface ContactOptions {
+    crn: string
+    relatedTo: string
+    category: string
+    type: string
+    team: string
+    staff: string
+}
+
+export const createContact = async (page: Page, options: ContactOptions) => {
+    await findContactsByCRN(page, options.crn)
+    await page.locator('input.btn', { hasText: 'Add Contact' }).first().click()
+    await expect(page).toHaveTitle('Add Contact Details')
+
+    await selectOption(page, '#addContactForm\\:RelatedTo', options.relatedTo)
+    await Promise.all([selectOption(page, '#addContactForm\\:ContactCategory', options.category), waitForAjax(page)])
+    await Promise.all([selectOption(page, '#addContactForm\\:ContactType', options.type), waitForAjax(page)])
+    await fillTime(page, '#addContactForm\\:StartTime', new Date())
+    await fillTime(page, '#addContactForm\\:EndTime', new Date(new Date().getTime() + 60000))
+    await Promise.all([selectOption(page, '#addContactForm\\:TransferToTeam', options.team), waitForAjax(page)])
+    await Promise.all([selectOption(page, '#addContactForm\\:Location'), waitForAjax(page)])
+    await Promise.all([selectOption(page, '#addContactForm\\:TransferToOfficer', options.staff), waitForAjax(page)])
+    await page.locator('#addContactForm\\:saveButton').click()
+    await expect(page).toHaveTitle(/Contact List/)
+}
+
+export const createInitialAppointment = async (page: Page, crn: string, eventNumber: string) =>
+    createContact(page, {
+        crn: crn,
+        relatedTo: `Event ${eventNumber} - ORA Community Order (6 Months)`,
+        category: 'All/Always',
+        type: 'Initial Appointment (NS)',
+        team: 'Unallocated Team(N03)',
+        staff: 'Unallocated',
```

Comment:
Would it make sense to move this to a `contacts` section in the [test-data](https://github.com/ministryofjustice/hmpps-probation-integration-e2e-tests/blob/main/test-data/environments/common.ts), to avoid hard-coding reference data in the tests?